temba/channels/models.py

```diff
@@ -699,6 +726,8 @@ def is_delegate_caller(self):
     def get_ivr_client(self):
         if self.channel_type == Channel.TYPE_TWILIO:
             return self.org.get_twilio_client()
+        if self.channel_type == Channel.TYPE_TWIML:
+            return self.org.get_twiml_client()
         if self.channel_type == Channel.TYPE_VERBOICE:
             return self.org.get_verboice_client()
```

Comment:
These really feel out of place on the org, especially so that there are now three of them. Seems the channel itself should be responsible for returning a valid ivr client.